src/content/libraries/async/async-await.md

```diff
@@ -306,11 +302,11 @@ void main() {
 
 ```plaintext
 Fetching user order...
-Your order is: Instance of '_Future<String>'
+Your order is: Instance of 'Future<String>'
 ```
 
-</div>
-<div class="col-sm">
+As shown in following two examples,
+resembles synchronous code.
```

Comment:
what does "resembles" mean here? As in it looks like synchronous? Maybe we can say that it "operates" equivalent to synchronous code?